opcpa_tpr_config/xpm_prog.py

```diff
@@ -0,0 +1,209 @@
+import argparse
+import itertools
+
+import numpy as np
+from psdaq.cas.pvedit import Pv
+from psdaq.seq.seq import Branch, ControlRequest, FixedRateSync
```

Comment:
I don't quite understand the behavior of these functions. so I'm just checking for typos and consistency.

I probably need to actually import psdaq proper and take a look

Reply:
`psdaq` isn't a standard import on `pcds_conda`. Probably in `dev_conda`, no?

Reply:
No, `psdaq` is kind of its own thing. Our `pcds_conda` environments import a very minimal portion of `psdaq` because it's a pretty heavy (and specific) environment. See here: https://github.com/pcdshub/pcds-envs/blob/master/envs/pcds/conda-packages.txt#L64

Because I need access to the tools that are in `psdaq`, I've simply updated my `PYTHONPATH` variable to include a path to a recent `psdaq` installation. This feels a little dirty, but it gets the job done. It seems the way that the DAQ team is managing production releases of `psdaq` is by building in `tmoopr` and `rixopr` directories. See here: https://github.com/pcdshub/opcpa-tpr-config/pull/40/files#diff-00fec08c5dea0e0456c484062562962f2aa5a623b0ada858b58951d684240bc5R11

I do have [a GH issue](https://github.com/pcdshub/opcpa-tpr-config/issues/55) where I am considering whether or not I want to take the time to build our own release area with `psdaq` releases. I'm still on the fence about it.

Reply:
Hmm, I'll take the time to educate myself on `psdaq` and the sequence functions on my own, but I don't think I'll hold up this PR any longer on that notion.

Reply:
If you're interested the available instructions are described here: https://confluence.slac.stanford.edu/display/~weaver/Timing+Pattern+Generator+Sequence+Programming